app/main.py

```diff
@@ -1,3 +1,42 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: [int, float, "Distance"]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: [int, float, Distance]) -> Distance:
```

Reply:
can u plz explain me, why my code doesn’t working , when i use Distance without  " "?
 NameError: name 'Distance' is not defined

Reply:
try to use
```python
from __future__ import annotations
```
 to solve this problem

Reply:
great, thanks!